cypress/integration/EditOpeningPeriod.spec.ts

```diff
@@ -51,15 +53,20 @@ describe('User edits an opening period', () => {
       timeout: 5000,
     }).should('be.visible');
 
-    // Start editing by checking the old title
-    cy.get('[data-test=openingPeriodTitle]')
+    // Start editing by checking the finnish title
+    cy.get('[data-test=openingPeriodTitle-fi]')
```

Comment:
The complexity of naming paradigms applied within this single string is quite high 😅 . There's probably good reason for it though?

Reply:
Aah, good point. I could try: openingPeriodTitleFi

Reply:
or actually: opening-period-title-fi

Reply:
[x] changed to "opening-period-title-fi" etc.